addons/payment/tests/common.py

```diff
@@ -188,7 +188,7 @@ def _create_transaction(self, flow, sudo=True, **values):
 
     def _create_token(self, sudo=True, **values):
         default_values = {
-            'name': "XXXXXXXXXXXXXXX-2565 (TEST)",
+            'payment_details': "2565123445677899",
```

Comment:
```suggestion
            'payment_details': "1234 (TEST)",
```
Why not this? So that'd be in the same logic as the real acquirers (last 4 digits) and still have something else to show it's a test, and show it's working even with a `payment_details` longer than the 4 digits.